### PR TITLE
Add mp.keys

### PR DESCRIPTION
**Pull request by emre1702**
Add mp.keys

### CODE REVIEW

File: types/Mp.Keys.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/// <reference path="../index.d.ts" />
+
+declare interface MpKeys {
+	isUp( keyCode: number ): void;
```
Comment (nick-strohm) [inline types/Mp.Keys.d.ts]: Indentation is still wrong here

**Comment by emre1702** [comment]
@nick-strohm Now?

**Comment by emre1702** [comment]
unbinds syntax could be wrong
Discord-guys said unbind doesn't have the keyHold argument.

**Comment by nick-strohm** [comment]
Could you test, approve it and if needed commit changes? Would be very appreciated

**Pull request merged**